src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-policy/rgw-multisite-sync-policy.component.ts

```diff
@@ -59,6 +59,7 @@ export class RgwMultisiteSyncPolicyComponent extends ListWithDetails implements
     this.columns = [
       {
         prop: 'uniqueId',
+        isInvisible: true,
```

Comment:
whats the difference between being invisible and hidden ?

Reply:
`isHidden`
![Screenshot from 2024-09-23 12-56-16](https://github.com/user-attachments/assets/243bda14-816b-4863-b60a-0c57227518db)

`isInvisible`
![Screenshot from 2024-09-23 12-56-47](https://github.com/user-attachments/assets/c8b5a24f-a57a-4354-8c53-1342722dc7d2)

Reply:
Also since you and I know about this that this having unique id in filter section does not makes sense. so lets add that in commit too, thanks

Reply:
updated in PR description, will update in commit also, Thanks